qa/tasks/cephfs/test_admin.py

```diff
@@ -219,14 +219,26 @@ def is_required(index):
 
             if index % 3 == 0:
                 continue;
-            self.fs.mon_manager.raw_cluster_cmd('fs', 'required_client_features', self.fs.name, 'add', feature)
+            self.fs.required_client_features('add', feature)
             self.assertTrue(is_required(index))
 
             if index % 2 == 0:
                 continue;
-            self.fs.mon_manager.raw_cluster_cmd('fs', 'required_client_features', self.fs.name, 'rm', feature)
+            self.fs.required_client_features('rm', feature)
             self.assertFalse(is_required(index))
 
+    def test_required_client_feature_add_reserved(self):
+        """
+        That `ceph fs required_client_features X add reserved` fails.
+        """
+
+        try:
+            self.fs.required_client_features('add', 'reserved')
+        except CommandFailedError:
+            pass
```

Comment:
Check for `exitstatus` and pass? I think you could add a success test for `add reserved by id` and a fail test for `rm reserved` too.

Reply:
Done.